src/site/controllers/NonRoutableWebPageController.php

```diff
@@ -148,13 +144,11 @@ protected function getGithubIssueBody(): string {
     $xhp->setContext('ServerRequestInterface', $this->request);
     $html = await $xhp->asyncToString();
 
-    await $response->getBody()->writeAsync($html);
-    $response = $response
-      ->withStatus($this->getStatusCode())
-      ->withHeader(
-        'Cache-Control',
-        vec['max-age=60'],
-      ); // enough for pre-fetching :)
+    $response = Response\html($html)
```

Comment:
`Response\xhp($xhp)` would be nice, since `$xhp->asyncToString()` returns an `HTMLString` and not a "normal" string.

Reply:
`Response\xhp` then would return an Awaitable, which seems inconsistent with other response factories such as `html`, `xml`, and `text`.

also, the point of `html` and other is that they add content type http header, with `xhp` we would be unable to do so, since we don't know the content type, it might be html, xml, or something else.